### ISSUE TITLE
ProvidedJS requires OS dependent path separator

### ISSUE THREAD

**Issue by rpiaggio** [scala-js/scala-js#2358]
In order to include `src/main/resources/js/dependency.js` on a build that is ran in Windows, we have to specify `ProvidedJS / "js\\dependency.js"`. 

`ProvidedJS / "js/dependency.js"` does not work. It results in the following error:

```
> show resolvedJSDependencies
[trace] Stack trace suppressed: run last compile:resolvedJSDependencies for the full output.
[error] (compile:resolvedJSDependencies) org.scalajs.core.tools.jsdep.JSLibResolveException: Some references to JS libraries could not be resolved:
[error] - Missing JS library: js/dependency.js
```

since the `String` returned by `FileVirtualJSFile.relativePath` contains a backslash.

**Comment by sjrd** [comment]
Thanks for the report!

**Comment by sjrd** [comment]
I am pretty sure that [this line](https://github.com/scala-js/scala-js/blob/3d894f2d380a7356f66c502550c8a479dc312a5a/sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala#L345) is responsible. We should normalize `relPath` to always using `/`. I'll do that when I'm on a Windows machine, hopefully tonight.

**Comment by rpiaggio** [comment]
Thank you very much for the fix! Is there an expected release date for 0.6.9?

**Comment by sjrd** [comment]
This week.
